functions/common/debezium-autoconfigure/README.adoc

```diff
@@ -0,0 +1,280 @@
+# Debezium Auto-Configuration
+
+This module provides a generic https://debezium.io/documentation/reference/2.2/development/engine.html[DebeziumEngine.Builder] auto-configuration that can be reused and composed in other applications.
+
+NOTE: The `DebeziumEngine` doesn't not required Kafka or Kafka Connect as it runs embedded inside your application. This approach though comes with some delivery guarantee limitations as explained https://debezium.io/documentation/reference/2.2/development/engine.html#_handling_failures[here].
+
+The `Debezium Engine` is a https://en.wikipedia.org/wiki/Change_data_capture[Change Data Capture] (CDC) utility, that allows *capturing* database change events and process them with custom `java.util.Consumer` or `io.debezium.engine.ChangeConsumer` event handler implementations.
```

Comment:
Why is everything below here now italicized? I think the `#` in the link above kicked that on.

Reply:
I think this is just this text render bug. If you go to File View, everything looks OK.

Reply:
Escapted the # in engine.html$$#$$_handling_failures 
Hopefully this will help.